### DISCUSSION TITLE
Idle Queues Causing High Memory Usage

### DISCUSSION THREAD

**Discussion by Juliesancho** [rabbitmq/rabbitmq-server#11182] [Questions]
Hi Good Day, 
We are using RabbitMQ 3.11.9, We have a high memory usage almost 90% of the RAM, 
We got 35k+ idle queues, We would like to find out why there are idles queues, and how the others (clients/users/companies) handled idle queues.

**Comment by michaelklishin** [comment]
There is very little that can be suggested given a two sentence problem description and no [relevant monitoring data](https://www.rabbitmq.com/docs/monitoring) or at least some relevant `rabbitmq-diagnostics`-provided description of the state of the node.

RabbitMQ 3.11 is [out of community support](https://www.rabbitmq.com/release-information). The core team will [not provide any guidance to non-contributors](https://github.com/rabbitmq/rabbitmq-server/blob/main/COMMUNITY_SUPPORT.md) on release series other than the latest.

We cannot know why there are idle queues in the system. Perhaps some queues are naturally not used much, or are leaked by applications. Perhaps they should use [queue TTL](https://www.rabbitmq.com/docs/ttl#queue-ttl), (some) be [exclusive or auto delete](https://www.rabbitmq.com/docs/queues#temporary-queues), or be explicitly deleted by applications when they are not needed any more.

Some relevant doc guides:

 * [Reasoning About Memory Use](https://www.rabbitmq.com/docs/memory-use)
 * [Reducing CPU Usage](https://www.rabbitmq.com/docs/runtime#cpu-reduce-idle-usage) (on mostly idle systems) and [management plugin stats collection frequency](https://www.rabbitmq.com/docs/management#statistics-interval)

Please take it from here.
